platform/android/src/android_renderer_frontend.cpp

```diff
@@ -73,8 +74,11 @@ void AndroidRendererFrontend::setObserver(RendererObserver& observer) {
 }
 
 void AndroidRendererFrontend::update(std::shared_ptr<UpdateParameters> params) {
-    mapRenderer.update(std::move(params));
-    mapRenderer.requestRender();
+    updateAsyncTask = std::make_unique<util::AsyncTask>([this, params]() {
```

Comment:
Can we just initialise the async task once in the constructor and call send here? You'll need to store the shared ptr to the update parameters as a member variable as well.

Reply:
@ivovandongen Thanks for pointing it out! I had thought `AsyncTask` is single use only, which was wrong. I'll change.